### ISSUE TITLE
bug: ion-side-menu-content does not adjust switching portrait/landscape mode

### ISSUE THREAD

**Issue by beatgrabe** [driftyco/ionic#5214]
My application does not adjust the `ion-side-menu-content` while switching between portrait and landscape mode. I could observe this issue running the application on an iPad with `expose-aside-when="large"` with an permanent open `ion-side-menu`. So starting the application in landscape keeps my content `width` on portrait mode, but it should be decreased. Starting in portrait mode and switching to landscape should increase the `width` of `ion-side-menu-content`. Apparently it does not work!

Futher information:
**ionic**: 1.2.4.
**cordova**: 5.4.1
**cordova-ios**: 3.9.2

**Viewport:**

```
<meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width"/>
```

**Comment by lordgreg** [comment]
I'm having the same issue. What I'll currently do is, watch on resize or rotation event, and manually recalucalte ion-content where I find ion-side-menu-content.

**Comment by beat-at-hartwig** [comment]
Nice hint! That's what i have done so far:

    window.addEventListener( "orientationchange", recalculateWidth );
    function recalculateWidth( event ) {
        var amount = parseInt( angular.element( "ion-side-menu" )[ 0 ].style.width, 10 );
        var total = parseInt( window.innerWidth, 10 );
        angular.element( "ion-side-menu-content" )[ 0 ].style.width = total - amount + "px";
    }

**Comment by lordgreg** [comment]
@beat-at-hartwig - just updated your code a bit. When switching different sites with such a content, you can have more than one ion-side-menu-content. That's why I suggest you create .length variable and use the for loop and update the width of all occurrences of ion-side-menu-content that you find.

:+1:

**Comment by EzheG** [comment]
+1

**Comment by leonardobazico** [comment]
+1

**Comment by leonardobazico** [comment]
+1

**Comment by GenesioValois** [comment]
+1

**Comment by felly** [comment]
Maybe a duplicate of #4898?

**Comment by beat-at-hartwig** [comment]
Good to know. Have not seen it yet. Thanks. I will give it a try!

**Comment by beatgrabe** [comment]
Can be closed due to fixes and merges in  [#4908](https://github.com/driftyco/ionic/pull/4908) and [#4898](https://github.com/driftyco/ionic/issues/4898)!
